microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -96,6 +96,31 @@ def verify_cloud_hypervisor_live_migration_tests(
             result, environment, "integration-live-migration", hypervisor
         )
 
+    @TestCaseMetadata(
+        description="""
+            Runs cloud-hypervisor live migration tests.
+        """,
+        priority=3,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                core_count=search_space.IntRange(min=16),
```

Comment:
1. Why specify those requirements? The requirements should be minimized to maximize tested scope. If there are special reasons on cpu and memory, add it in description part.

Reply:
This we will manage in separate PR of CH. I will remove this as there is no specific requirements.